x/tokenomics/keeper/token_logic_modules_test.go

```diff
@@ -399,14 +399,15 @@ func TestProcessTokenLogicModules_TLMGlobalMint_Valid_MintDistributionCorrect(t
 
```

Comment:
Depending on the outcome of this: https://github.com/pokt-network/poktroll/pull/878/files#r1810783728

Can you add a dedicated (short) test for it?

Reply:
`TestProcessTokenLogicModules_TLMBurnEqualsMint_Valid_SupplierExceedsMaxClaimableAmount` has been adjusted and describes exactly that even if the claim is over max claimable, an event is emitted for over-charging and max claimable is effective.

Will add a TODO_TEST for edge values though.